src/functionalTest/java/uk/gov/hmcts/reform/notificationservice/RestAssuredHelper.java

```diff
@@ -45,7 +45,9 @@ static String s2sSignIn(String microservice) {
             .post("/lease")
             .andReturn();
 
-        assertThat(response.getStatusCode()).isEqualTo(200);
+        assertThat(response.getStatusCode())
+            .as("OTP leased successfully")
```

Comment:
otp is sent to this endpoint, not received from it.

Reply:
what we get here is s2s token I think

Reply:
lease means contract or some sort of grant. in super short way. so looks like sentence is ok. feel free to suggest any sentence that may be suitable/more understandable for us :slightly_smiling_face:

Reply:
method is called `s2sSignIn` so maybe 'Signed in to S2S successfully'.